pds/stubs.go

```diff
@@ -804,7 +804,6 @@ func (s *Server) RegisterHandlersComAtproto(e *echo.Echo) error {
 	e.POST("/xrpc/com.atproto.admin.takeModerationAction", s.HandleComAtprotoAdminTakeModerationAction)
 	e.POST("/xrpc/com.atproto.admin.updateAccountEmail", s.HandleComAtprotoAdminUpdateAccountEmail)
 	e.POST("/xrpc/com.atproto.admin.updateAccountHandle", s.HandleComAtprotoAdminUpdateAccountHandle)
-	e.GET("/xrpc/com.atproto.identity.resolveHandle", s.HandleComAtprotoIdentityResolveHandle)
```

Comment:
This is still a real lexicon endpoint, used by clients to resolve handlers without going out to the network themselves.

The PDS isn't "real" so don't need to implement it, but seems like it should at least stay a stub with an HTTP "not implemented" error.

Reply:
Ah, I missed that, thought we did away with it entirely for some reason